tests/parm/nems.configure.hafs_atm_wav.IN

```diff
@@ -76,21 +76,24 @@ WAV_attributes::
   Diagnostic = 0
   OverwriteSlice = false
   merge_import = .true.
+  mask_value_water = 1
```

Comment:
I want to confirm that the mask_value_water means the valid value on water points, is it correct?

Reply:
This will make the current WW3 cap set the mask value for water to 1.